whelktool/scripts/cleanups/2024/08/lxl-4500-licenses.groovy

```diff
@@ -0,0 +1,72 @@
+String where = "collection = 'bib' and deleted = false and data#>>'{@graph,1,usageAndAccessPolicy}' like '%creativecommons.org%' "
+
+selectBySqlWhere(where) { data ->
+    Map instance = data.graph[1]
+
+    if (! instance["usageAndAccessPolicy"] instanceof List)
+        instance.put("usageAndAccessPolicy", [instance["usageAndAccessPolicy"]])
+
+    //System.out.println("BEFORE: ${instance["usageAndAccessPolicy"]}")
+
+    boolean changed = false
+    instance["usageAndAccessPolicy"].each { usage ->
+
+        if (usage["uri"] != null) {
+            usage["uri"].removeAll { uri ->
```

Comment:
This could just be `each` since usage is cleared anyway?

Reply:
Well yes.. :) But I dont want to change and have to test again if not necessary.